lib/vae/channels/contact_channel.ex

```diff
@@ -52,4 +75,27 @@ defmodule Vae.ContactChannel do
       []
     end
   end
+
+  defp delegate_message(body = %{"contact_delegate" => "on"}) do
```

Comment:
Make a function to generate template, because the only things that change is the `ReplyTo` and `TemplateId` compared to vae_recap_message, no ?